model-optimizer/unit_tests/mo/frontend_ngraph_test.py

```diff
@@ -1,11 +1,29 @@
 # Copyright (C) 2018-2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+import logging as log
 import os
 import subprocess
 import sys
+import unittest
+from unittest.mock import patch
+
+from mo.subprocess_main import setup_env, subprocess_main
+
+import pytest
+
+
+class TestNoInferenceEngine(unittest.TestCase):
+    @patch('mo.utils.find_ie_version.find_ie_version')
+    def test_no_ie_ngraph(self, mock_find):
+        mock_find.return_value = False
+        with pytest.raises(SystemExit) as e, self.assertLogs(log.getLogger(), level="ERROR") as cm:
+            subprocess_main()
```

Comment:
does it depend on what machine w/o installed IE modules this test runs?

Reply:
No, it doesn't depend on this - it 'patches' find_ie_version' function to force returning 'False'. No matter if IE is installed or not